pkg/collector/nakadi_collector.go

```diff
@@ -72,14 +71,39 @@ func NewNakadiCollector(_ context.Context, nakadi nakadi.Nakadi, hpa *autoscalin
 		return nil, fmt.Errorf("metric-type must be either '%s' or '%s', was '%s'", nakadiMetricTypeConsumerLagSeconds, nakadiMetricTypeUnconsumedEvents, metricType)
 	}
 
+	// Either subscription-id or filtering via owning-application,
+	// event-types, and consumer-group is supported. If all are defined
+	// then subscription-id is used.
```

Comment:
"If all are defined then subscription-id is used". Can you elaborate on this? I'm assuming if we already have the `susbscription-id` defined then the rest of the parameters will not be used, so we can ignore them, I guess?

Reply:
Yes, the other fields, except `subscription-id` are used for find a `subscription-id`. If `subscription-id` is already defined the other fields are not used for filtering.

I have updated the comment a bit to clarify this.

Reply:
Thanks!